docker-compose.yaml

```diff
@@ -54,6 +54,11 @@ services:
       - CLIENT_ASSERTION=${CLIENT_ASSERTION}
       - TENANT_ID=${TENANT_ID}
       - AUTH_TYPE=${AUTH_TYPE}
+      - AAD_EH_SAS_POLICY=${AAD_EH_SAS_POLICY}
```

Comment:
Please add a comment explaining what these mean in their full form. 
#Azure Active Directory Enterprise Helpdesk Software as a Service Policy

It doesn't have to happen immediately since we have demo tomorrow.